### PR TITLE
feat(mailer): configure nodemailer with the right smtp server

### PR DESCRIPTION
**Pull request by dryhten**
re #673

### CODE REVIEW

File: config/mailer.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
 module.exports = {
   transport: {
-    host: 'smtp.ethereal.email',
-    port: 587,
-    auth: {
-      user: 'g3jjujasexmtz4dl@ethereal.email',
-      pass: 'QF2bP2FcHa5cVCVPPT',
-    },
+    host: 'localhost',
```
Comment (diversemix) [inline config/mailer.js]: Do we want to configure this differently on a per-environment basis? Or are these settings ok for all? @giorgiosironi - what would you recommend?

**Comment by dryhten** [comment]
This needs to be tested in a production-like environment before removing the `WIP` tag.

**Comment by diversemix** [comment]
@dryhten ... I tested this on `demo` with the "mail" command and I did receive the mail.

**Comment by dryhten** [comment]
@diversemix I did that a couple of times as well. I was thinking we need to test from the app that `nodemailer` will actually send the mail to `postfix` with these credentials. In theory it should work, postfix has a `mynetworks` config for trusted networks which is set to `127.0.0.0/8` so everything coming from localhost should be trusted and no authentication should be required, but you never know what might go wrong.

**Pull request merged**